multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -421,19 +425,20 @@ def process_one_sample(
         ret = {}
         column_start = 0
         if self.prefix.lower().startswith(MMLAB_MODELS):
-            for per_col_name, per_col_content in image_paths.items():
+            for per_col_name, per_col_content in image_features.items():
                 if is_rois_input(per_col_content):
                     rois = np.array(per_col_content)
                     # TODO: add gt masks
                     mm_data["ann_info"] = dict(bboxes=rois[:, :4], labels=rois[:, 4], masks=[])
                 else:
-                    with PIL.Image.open(per_col_content[0]) as img:
-                        mm_data["img_info"] = dict(filename=per_col_content[0], height=img.height, width=img.width)
+                    if feature_modalities.get(per_col_name) == IMAGE_PATH:
```

Comment:
What if detection uses image bytearray?

Reply:
It is currently not supported in this PR. `mmdet` doesn't directly support load image from bytearray, so it is a bit more work to add the support. I can add in the next PR.